test/data/testsuite.css

```diff
@@ -142,3 +142,7 @@ section { background:#f0f; display:block; }
 	padding: 0;
 	margin: 0;
 }
+
+.border-box, .border-box * {
+	box-sizing: border-box;
+}
```

Comment:
That's a total nit but why not:
```css
.border-box {
	box-sizing: border-box;
}
.border-box * {
	box-sizing: inherit;
}
```
? It's easier to override the behavior in sub-trees then.

Reply:
Never occurred to me, but the point of this is to explicitly set `border-box` on all of the elements. Allow them to overridden seems like it defeats the purpose.